.github/workflows/ci.yaml

```diff
@@ -75,20 +52,63 @@ jobs:
           retention-days: 5
 
       - name: Java Linter
-        run: bazel --batch run //:google_java_format_verification
+        run: bazelisk --batch run //:google_java_format_verification
```

Comment:
I think we can remove Java linter as well if it's already included in the Maven build.

Reply:
Removed.